### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ScreenReader: Voice Over is not announcing the level for the non editable text box,edit box and drop down combo box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6155]
**Bug Description**:
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS4.1.2; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details**:
Application name: Azure Data Studio 
App Version: 1.9.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: MacOS Mojave 10.14.5
**Additional Details**:
**Tool Name**: Voice Over
**MAS Reference**:MAS1.3.1,MAS4.1.2
**Repro Steps:**
1.Install the app Azure Data Studio.
2.Enable SQL server2019 preview.
3.Select table from the database.
4.Create External Table screen should open->Next button.
5.Create Database Master Key screen should open->click on next button.
6.Fill all the required details->click on next button.
7.Open NVDA and Observe the announcement.
**Actual**:
Screen Reader is not announcing the level for the non editable text box ,edit box, drop down combo box. It is announcing the selected text inside the text box.
**Expected**:
It should announce the level for non editable text box, edit box, drop down combo box.
**User Impact**:
User with disabilities will not be able to relate with the edit boxes that what input he or she needs to give in the related box.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference**:MAS1.3.1,MAS4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
**Attachment for Reference**:
[6155_AccessibilityA11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3325193/6155_AccessibilityA11y_AzureDataStudio_MapDataSrcObjects_ScreenReader.pptx)

**Comment by MaddyDev** [comment]
@Charles-Gagnon clarified this issue with the accessibility team and the issue is not with the announcing the level but label.
